go/oasis-net-runner/fixtures/default.go

```diff
@@ -221,12 +231,68 @@ func newDefaultFixture() (*oasis.NetworkFixture, error) {
 				fixture.ComputeWorkers[j].Runtimes = append(fixture.ComputeWorkers[j].Runtimes, rtIndex)
 			}
 			fixture.Clients[0].Runtimes = append(fixture.Clients[0].Runtimes, rtIndex)
+
+			// Runtime state paths to use to initialize the runtime with.
+			runtimeStatePath := runtimeStatePaths[i]
+			// Set workers runtime state.
+			for j := range fixture.ComputeWorkers {
+				fixture.ComputeWorkers[j].RuntimeStatePaths[i] = runtimeStatePath
+			}
+			if runtimeStatePath == "" {
```

Comment:
This can probably be moved before the above for loop now that the `RuntimeStatePaths` is a map.

Reply:
yep, fixed